migrator/src/test/resources/simplelogger.properties

```diff
@@ -1,3 +1,3 @@
 org.slf4j.simpleLogger.defaultLogLevel=ERROR
-org.slf4j.simpleLogger.log.com.redhat.lightblue.migrator=ERROR
+org.slf4j.simpleLogger.log.com.redhat.lightblue.migrator=DEBUG
```

Comment:
I assume this was for testing purposes, but we probably don't actually want it this way.